starter-code/src/viking.js

```diff
@@ -1,11 +1,126 @@
 // Soldier
-function Soldier () {}
+function Soldier (health, strength) {
+  this.health = health;
+  this.strength = strength;
+}
+
+Soldier.prototype.attack = function() {
+  return this.strength;
+}
+
+Soldier.prototype.receiveDamage = function(damage) {
+  this.health= this.health - damage;
+}
```

Comment:
Esto se refactoriza en `this.health -= damage` algo más elegante y reducido, pero no está mal como ya escribisteis.